NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/RemoveMember.java

```diff
@@ -62,19 +61,16 @@ public boolean execute(CommandSender sender, String[] args) {
 		boolean allowed = false;
 		switch (toBeRemoved){ // depending on the type the executor wants to add the player to
 		case MEMBERS:
-			allowed = perm.isAccessible(t, PermissionType.MEMBERS);
+			allowed = gm.hasAccess(group.getName(), uuid, PermissionType.getPermission("MEMBERS"));
```

Comment:
I was a bit worried about other plugins using outdated group references, because they often store a direct reference to the group and don't update when the group is invalidated (which happens every time something about the group is changed on another shard). I guess if we make sure to check for invalidation etc. here I could add a method using the group instead of the name.